cmd/kube-apiserver/app/server_test.go

```diff
@@ -100,7 +100,7 @@ func TestUpdateEtcdOverrides(t *testing.T) {
 	}
 
 	for _, test := range testCases {
-		newEtcd := func(serverList []string, _ runtime.NegotiatedSerializer, _, _, _ string, _ bool) (storage.Interface, error) {
+		newEtcd := func(serverList []string, _ runtime.NegotiatedSerializer, _, _, _ string, _, _, _ string, _ bool) (storage.Interface, error) {
```

Comment:
Might be worth testing that the parameters are properly passed to this function.